app/controllers/api/v1/referees_controller.rb

```diff
@@ -8,7 +8,8 @@ class RefereesController < ::ApplicationController
       layout false
 
       def index
-        @referees = Services::FilterReferees.new(search_params).filter
+        referee_ids = Services::FilterReferees.new(search_params).filter
+        @referees = Referee.where(id: referee_ids)
```

Comment:
Please add the `.includes(:certifications, :national_governing_bodies)` here too to not have the serializer do lazy loads again to serialize the certifications and NGBs. For the 270 referees in my local database that’s a difference of having 6 database queries instead of ~550.